lib/parse_date.rb

```diff
@@ -72,7 +72,7 @@ def self.range_array(first_year, last_year)
     return [] unless last_year || first_year
     return [first_year] if last_year.nil? && first_year
     return [last_year] if first_year.nil? && last_year
-    raise(StandardError, "unable to create year range array from #{first_year}, #{last_year}") unless
+    raise(ParseDate::Error, "unable to create year range array from #{first_year}, #{last_year}") unless
```

Comment:
we want this gem to throw errors specific to this gem - this is the one occurrence where that was not the case.